app/mailers/billing_subscription_mailer.rb

```diff
@@ -3,7 +3,6 @@ class BillingSubscriptionMailer < BaseMailer
   def failed_notice(np_id)
     @nonprofit = Nonprofit.find(np_id)
     @billing_subscription = @nonprofit.billing_subscription
-    @card = @nonprofit.active_card
```

Comment:
So you've deleted this line but what is this mailer used for? What about the view associated with the mailer action, does this change break it? Or do we even need the mailer or view?